test_sharding/run_standard_tests_with_sharding.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
can you add a readmit ile to this folder explaining briefly what the scripts do?

Reply:
done